rust/agents/scraper/migration/src/m20220805_000005_create_table_delivered_message.rs

```diff
@@ -3,7 +3,6 @@ use sea_orm_migration::prelude::*;
 use crate::l20220805_types::*;
```

Comment:
It's because of sea-orm spec. Date is the original migration creation, thing is we keep updating them because as of right now we are not actually in a state where we need to properly maintain a set of migrations that are append only as we are willing to re-create the db. Ideally I would change the date on all the files every time I make any change, but that would cause other challenges and just make a lot of file renames. The `l` is library and the `m` is migration.

https://www.sea-ql.org/SeaORM/docs/migration/setting-up-migration/